tests/lib/totp.js

```diff
@@ -0,0 +1,143 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'intern!tdd',
+  'intern/chai!assert',
+  'tests/addons/environment',
+  'tests/addons/sinon',
+  'node_modules/otplib/otplib-browser'
+], function (tdd, assert, Environment, sinon, otplib) {
+
+  with (tdd) {
+    suite('totp', function () {
+      var account;
+      var accountHelper;
+      var respond;
+      var client;
+      var RequestMocks;
+      var env = new Environment();
+      var xhr;
+      var xhrOpen;
+      var xhrSend;
+
+      beforeEach(function () {
+        env = new Environment();
+        accountHelper = env.accountHelper;
+        respond = env.respond;
+        client = env.client;
+        RequestMocks = env.RequestMocks;
+
+        return accountHelper.newVerifiedAccount()
+          .then(function (newAccount) {
+            account = newAccount;
```

Comment:
nit: this local variable seems a little redundant, we could just name the parameter `account`.

Reply:
This is actually setting the value for the global [account object](https://github.com/mozilla/fxa-js-client/blob/ec00e34a8438a2e15098d0bcfc961b65fb3bb6fe/tests/lib/totp.js#L15), which is reused in each test case.

Reply:
Oh right, of course :-)